src/config/ConfigManager.cpp

```diff
@@ -65,16 +67,16 @@ std::string CConfigManager::removeBeginEndSpacesTabs(std::string str) {
 }
 
 void CConfigManager::parseLine(std::string& line) {
-    // first check if its not a comment
+    // First check if it's not a comment
     const auto COMMENTSTART = line.find_first_of('#');
     if (COMMENTSTART == 0)
         return;
 
-    // now, cut the comment off
+    // Remove comment from string
     if (COMMENTSTART != std::string::npos)
         line = line.substr(0, COMMENTSTART);
 
-    // remove shit at the beginning
+    // Strip line
```

Comment:
why these 3?

Reply:
As you wanted, I revert the previous change concerning the two first comments. However, a rude word in a code base isn't something you want, and "strip" simply seems to better explain what's happening.